### PR TITLE
Misc QoL Updates

### PR DESCRIPTION
**Pull request by Huckleberry87**
-Raised Hummingbird up 2 feet, to avoid respawn explosions. 
-Swapped out BN Aviator billboard for Patreon Billboard. 
-Added another wreck recovery East of ACAV (marked on map) 
-No more wrecks for ARVN
-Remove BH Shed, to avoid respawn explosions.
-Added SOG Camera saving to disk.
-Changed [REDACTED] to allow [REDACTED]

### CODE REVIEW

**Comment by dijksterhuis** [comment]
i'll try to pull and review locally tomorrow

**Comment by dijksterhuis** [comment]
> Changed [REDACTED] to allow [REDACTED]

🤣

File: maps/cam_lao_nam/mission.sqm [hunk 1]
```diff
@@ -4969,7 +4969,7 @@ class Mission
 										class data
 										{
 											singleType="STRING";
-											value="custom\billboards\gh_bb.paa";
+											value="custom\billboards\supporter_sign.paa";
```
Comment (dijksterhuis) [inline maps/cam_lao_nam/mission.sqm:4972]: paa path here doesn't match the paa path of added file -- `mission/custom/billboards/Supporter_SIgn.paa`

is that an issue or does it work fine? (i'll find out tomorrow anyway).

File: mission/config/subconfigs/vehicle_respawn_info.hpp [hunk 2]
```diff
@@ -736,7 +736,7 @@ class spawn_point_types {
 	class arvn_grnd_firesupport_light_custom {
```
Comment (dijksterhuis) [inline mission/config/subconfigs/vehicle_respawn_info.hpp:736]: no extra arvn statics spawner? John --> 😢

Comment (Huckleberry87) [reply mission/config/subconfigs/vehicle_respawn_info.hpp:736]: uhhh.... next update

File: maps/cam_lao_nam/mission.sqm (hunk 1 above)
Comment (Huckleberry87) [reply maps/cam_lao_nam/mission.sqm:4972]: it worked on my end.

File: mission/config/subconfigs/vehicle_respawn_info.hpp (hunk 2 above)
Comment (dijksterhuis) [reply mission/config/subconfigs/vehicle_respawn_info.hpp:736]: this PR, or a separate PR?

Comment (Huckleberry87) [reply mission/config/subconfigs/vehicle_respawn_info.hpp:736]: separate that I will do another time.

Comment (Huckleberry87) [reply mission/config/subconfigs/vehicle_respawn_info.hpp:736]: forgot your answers from a few days ago.

Comment (dijksterhuis) [reply mission/config/subconfigs/vehicle_respawn_info.hpp:736]: ok.

(btw i'm loving the git log giving me notifications about PR things without having to look at my email 🎉 )

**Comment by dijksterhuis** [comment]
code LGTM but will checkout and test locally just as a double check cos it's 5am my time

File: maps/cam_lao_nam/mission.sqm (hunk 1 above)
Comment (dijksterhuis) [reply maps/cam_lao_nam/mission.sqm:4972]: yeah looks fine

File: maps/cam_lao_nam/mission.sqm [hunk 3]
```diff
@@ -41073,13 +41075,83 @@ class Mission
...
+							{
+								position[]={11926.36,70.261833,14373.789};
+							};
+							side="Empty";
+							class Attributes
+							{
+							};
+							id=18741;
+							type="Land_HelipadEmpty_F";
+						};
```
Comment (dijksterhuis) [inline maps/cam_lao_nam/mission.sqm:41147]: not fixed the gia lam issue 😢 

not sure what the deal with this is then as everything is attached and placed correctly

might need further/deeper script investigation

**Comment by dijksterhuis** [comment]
reviewed, good to go. gia lam master arms seem to be something not editor based. not sure what's happening there, but it's not a calamitous issue.

**Comment by dijksterhuis** [comment]
i think you also got the mike force lamp that was hovering slightly above ground (or i was imagining things due to lack of sleep, it was one of the two)

**Pull request merged**